src/config/model.rs

```diff
@@ -68,7 +68,7 @@ impl Account {
 
     pub fn smtp_creds(&self) -> Result<SmtpCredentials> {
         let passwd = run_cmd(&self.smtp_passwd_cmd).chain_err(|| "Cannot run SMTP passwd cmd")?;
-        let passwd = passwd.trim_end_matches("\n").to_owned();
+        let passwd = passwd.trim_end().to_owned();
```

Comment:
With `trim_end` you also remove spaces, which are *in theory* valid chars for a password. I admit that *in practice* nobody uses this char in his password, but still. What do you think about:

```rust
let passwd = passwd
    .trim_end_matches(|c| c == '\r' || c == '\n')
    .to_owned();
```